batch_processing.sh

```diff
@@ -260,7 +260,7 @@ echo "Ran on:          `uname -nsr`"
 echo "Duration:        $(($runtime / 3600))hrs $((($runtime / 60) % 60))min $(($runtime % 60))sec"
 echo "---"
 # The file `test_batch_processing.py` will output tested values when run as a script
-"$SCT_DIR"/python/envs/venv_sct/bin/python $SCT_DIR/unit_testing/batch_processing/test_batch_processing.py
+"$SCT_DIR"/python/envs/venv_sct/bin/python **/test_batch_processing.py
```

Comment:
I don't think you can do this. You can simply replace `unit_testing` with `testing`.

Reply:
Oops. It works for me locally, but that's because [zsh supports recursive globbing](https://lorenzod8n.wordpress.com/2007/05/10/recursive-globbing-in-zsh/). I forgot this wasn't a general `sh` thing.